blog/poc3.md

```diff
@@ -0,0 +1,127 @@
+
+MRU is an experimental feature in current POC3 testnet and is still  undergoing changes.
+
+## Drop box
+
+We mentioned that thanks to manifests, directory trees and collections represented by manifests can be mapped uniquely. This does not only allow upload and download into directory but also use FUSE integration to mount a swarm manifest as a local file system.
+Due to the fact that swarm retrieval is on-demand, Swarm FUSE mounts are readable with no latency. Swarm FUSE supports file system write operations (creating, appending, updating files) by maintaining the collection swarm hash as an internal state.
+Combining FUSE with MRU, on each change we can issue a mutable resource update, this makes it possible to think of FUSE to sync your entire home folder between devices, virtually serving as the backend to a decentralised drop box.
+
+Swarm POC3 supports encryption so protecting your private data has never been easier. Once Access Control Trees are implemented, we will have a fully fledged solution for 'sharing' a subdirectory, a feature well known from Dropbox or Google Drive. Combining with PSS the 'share' can be sent in a notification to the recipient.
+
+Both [FUSE](http://swarm-guide.readthedocs.io/en/usage.html#FUSE) and [encryption](http://swarm-guide.readthedocs.io/en/usage.html#Encryption) are fully functional in POC3.
+
+# The year ahead
+
+Just as the past year, these coming 10+ month will be very very exciting and challenging. We are on target delivering Swarm POC4 (production beta prerelease) in the first half of 2019.
+
+As still part of the POC3 series, we are planning to switch on a revamped SWAP accounting for chunk deliveries.
+Implementing access control and authentication for swarm content is high on our priority list.
+The first few releases will also address some technical debt accumulated as well as optimisations of retrieval and syncing.
+After rounding up [the swap swear and swindle framework for decentralised service economies](https://github.com/ethersphere/swap-swear-and-swindle/tree/master) (see the [3rd orange paper]( https://www.sharelatex.com/1452913241cqmzrpfpjkym) and a [talk](https://youtu.be/Bn65-bI-S1o) and the [implementation](https://github.com/ethersphere/swap-swear-and-swindle/tree/master)), we apply it to the storage insurance using the [crash proof of custody scheme](https://swarm-gateways.net/bzz:/theswarm.eth/ethersphere/orange-papers/1/sw^3.pdf).
+
+We are implementing two layers of erasure coding for redundancy, one is primarily against dropout and network switch issues to guarantee low latency retrieval. Another one is part of the two tiered insurance system, where stakeless farmers get positive rewards for proof of custody while staked insurers continually perform scan and repair on highly redundantly coded chunks potentially suffering punitive measures as well as needing to pay compensatory insurance to users if integrity of insured content is compromised.
```

Comment:
Another one is part of the two tiered insurance system. **S**takeless farmers get positive rewards for proof of custody. Staked insurers continually perform scan-and-repair on highly redundant **stored** **(coded sounded strange to me here).** chunks. Punitive measures may be taken as well as compensatory insurance fees levied against storers if integrity of insured content is compromised.